src/main/java/calculator/stringCal.java

```diff
@@ -0,0 +1,60 @@
+package calculator;
+
+public class stringCal {
+
+    //string을 반전시키는 함수
+    public static String reverseStr(String str) {
+        String reverse = "";
+        for (int i = str.length() - 1; i >= 0; i--) {
+            reverse = reverse + str.charAt(i);
+        }
+        return reverse;
+    }
+
+    public String numCal(String first, String second){
+        int carry = 0;
+        String result ="";
+        //각 자리수 비교하기 위해 min, max로 비교
+        int min_val = Integer.min(first.length(), second.length());
+        int max_val = Integer.max(first.length(), second.length());
```

Comment:
`Integer.parseInt()`로 문자열을 숫자로 변환할 수 있습니다
계산기 직접 구현하신 부분은 좋지만 지금 미션의 의도는 이게 아닌 것 같아요~~

Reply:
참고로 변수명은 카멜 케이스로 작성해주세요!

```suggestion

    public String numCal(String first, String second){
        int carry = 0;
        String result ="";
        //각 자리수 비교하기 위해 min, max로 비교
        int minVal = Integer.min(first.length(), second.length());
        int maxVal = Integer.max(first.length(), second.length());
```